src/acl/acl-control.ts

```diff
@@ -49,87 +45,100 @@ UI.aclControl.preventBrowserDropEvents = function (document) {
     'style',
     'min-width: 20em; padding: 1em; font-size: 120%; border-bottom: 0.1em solid red; margin-bottom: 2em;'
   )
 
-  var statusRow = table.appendChild(dom.createElement('tr'))
+  const statusRow = table.appendChild(dom.createElement('tr'))
 
-  var statusCell = statusRow.appendChild(dom.createElement('td'))
-  var statusBlock = statusCell.appendChild(dom.createElement('div'))
+  const statusCell = statusRow.appendChild(dom.createElement('td'))
+  const statusBlock = statusCell.appendChild(dom.createElement('div'))
   statusBlock.setAttribute('style', 'padding: 2em;')
-  var MainRow = table.appendChild(dom.createElement('tr'))
-  var box = MainRow.appendChild(dom.createElement('table'))
-  var bottomRow = table.appendChild(dom.createElement('tr'))
+  const MainRow = table.appendChild(dom.createElement('tr'))
+  const box: any = MainRow.appendChild(dom.createElement('table'))
+  const bottomRow = table.appendChild(dom.createElement('tr'))
 
   // A world button can be dragged to gve public access.
   // later, allow it to be pressed to make pubicly viewable?
-  var bottomLeftCell = bottomRow.appendChild(dom.createElement('td'))
-  // var bottomMiddleCell = bottomRow.appendChild(dom.createElement('td'))
-  var bottomRightCell = bottomRow.appendChild(dom.createElement('td'))
+  const bottomLeftCell = bottomRow.appendChild(dom.createElement('td'))
+  // const bottomMiddleCell = bottomRow.appendChild(dom.createElement('td'))
+  const bottomRightCell = bottomRow.appendChild(dom.createElement('td'))
 
-  // var publicAccessButton = bottomLeftCell.appendChild(UI.widgets.button(dom, UI.icons.iconBase + 'noun_98053.svg', 'Public'))
+  // const publicAccessButton = bottomLeftCell.appendChild(widgets.button(dom, icons.iconBase + 'noun_98053.svg', 'Public'))
 
   const bigButtonStyle =
     'border-radius: 0.3em; background-color: white; border: 0.1em solid #888;'
 
   // This is the main function which produces an editable access control.
   // There are two of these in all iff the defaults are separate
   //
-  function ACLControlEditable (box, doc, aclDoc, kb, options = {}) {
-    var ac = UI.acl.readACL(doc, aclDoc, kb, options.doingDefaults) // Note kb might not be normal one
-    var byCombo = UI.acl.ACLbyCombination(ac) // important to keep this, as removal will trigger bugs like https://github.com/solid/solid-panes/issues/193#issuecomment-549777549
-
-    var kToCombo = function (k) {
-      var y = ['Read', 'Append', 'Write', 'Control']
-      var combo = []
-      for (var i = 0; i < 4; i++) {
+  function ACLControlEditable (
+    box,
+    doc,
+    aclDoc,
+    kb,
+    options: {
+      doingDefaults?: boolean,
+      modify?: boolean
+    } = {}) {
+    const ac = readACL(doc, aclDoc, kb, options.doingDefaults) // Note kb might not be normal one
+    const byCombo = ACLbyCombination(ac) // important to keep this, as removal will trigger bugs like https://github.com/solid/solid-panes/issues/193#issuecomment-549777549
```

Comment:
Don't need "important to keep" notice .. no real reason for people noit to keep it.  It was removed once because of a misunderstanding.  (https://github.com/solid/solid-ui/commit/5afb141378f195f395f249258a7fd02768334ada#commitcomment-36205141 )